src/main/java/org/cyclonedx/gradle/utils/CycloneDxUtils.java

```diff
@@ -49,4 +57,43 @@ public static Version schemaVersion(String version) {
                 return DEFAULT_SCHEMA_VERSION;
         }
     }
+
+    public static void writeBom(
+            final Bom bom,
+            final File destination,
+            final String outputName,
+            final Version version,
+            final String formats) {
+
+        if (formats.equals("all") || formats.equals("json")) {
+            final File jsonFile = new File(destination, String.format("%s.json", outputName));
+            writeJSONBom(version, bom, jsonFile);
+        }
+
+        if (formats.equals("all") || formats.equals("xml")) {
+            final File xmlFile = new File(destination, String.format("%s.xml", outputName));
+            writeXmlBom(version, bom, xmlFile);
```

Comment:
Please validate generated BOM as it's done now https://github.com/CycloneDX/cyclonedx-gradle-plugin/blob/master/src/main/java/org/cyclonedx/gradle/CycloneDxTask.java#L830

Reply:
Validation is added.